src/Generators/Microsoft.Gen.AutoClient/Emitter.cs

```diff
@@ -489,4 +514,58 @@ private void EmitClassVariablesAndConstructor(RestApiType restApiType, string ht
         OutLn($"{optionsName} = autoClientOptions;");
         OutCloseBrace();
     }
+
+    private void EmitPathParameterValidation()
```

Comment:
Instead of generating this, could this be a static method in the M.E.EnumStrings package?

Reply:
I can do that, the package is quite empty right now, and I thought it was only for enums manipulation (or that it would go away). Does it still make sense and should I then just add a new static class with this helper?

Reply:
Sorry, I meant a utility method in M.AspNetCore.AutoClient. This has nothing to do with enum strings.

Reply:
Moved to `AutoClientUtilities.cs` (new file)